readme.md

```diff
@@ -38,65 +38,49 @@ $ npm -v
 
 * **Question:** What is the difference between forking and cloning a repository as opposed to just cloning a repository?
 
-* **Your Answer:**
-
----
+* **Your Answer:** A fork is a copy of a repository that allows you to experiment with changes without affecting the original project. Cloning makes a local repo that is not connected to the original repo. 
```

Comment:
Small point of clarification: cloning makes a local repo that _is_ connected to the repo that you cloned from. When you fork _and then_ clone, you end up with a local version of the repo that is not connected to the original.